lib/inputstreamhelper/__init__.py

```diff
@@ -223,8 +227,11 @@ def enable():
 
     def _inputstream_version(self):
         ''' Return the requested inputstream version '''
-        addon = Addon(self.inputstream_addon)
-        return to_unicode(addon.getAddonInfo('version'))
+        try:
+            addon = Addon(self.inputstream_addon)
+            return to_unicode(addon.getAddonInfo('version'))
```

Comment:
I would move the return out of the except. Even if we would never expect a RuntimeError in that line, it is always a good practice to keep the stuff in an except-block to the bare minimum as possible

Reply:
Edited.